src/components/utils/include/utils/timer.h

```diff
@@ -69,7 +69,8 @@ class Timer {
   /**
    * @brief Starts timer with specified timeout
    * @param timeout Timer timeout
-   * @param single_shot Shows needs to restart timer after timeout
+   * @param single_shot Shows if timer should be restarted automatically
```

Comment:
@anosach-luxoft, the description of the `single_shot` param looks strange to me. Single shot means that there will be no repeat at all. Please adjust comment.